compressor/filters/base.py

```diff
@@ -119,7 +119,9 @@ class CompilerFilter(FilterBase):
     """
     command = None
     options = ()
-    default_encoding = settings.FILE_CHARSET
+    default_encoding = (
+        settings.FILE_CHARSET if hasattr(settings, 'FILE_CHARSET') else 'utf-8'
```

Comment:
does that work? i just fired up a console and the `hasattr` returns true on django 2.2. You could experiment with  `settings.is_overridden`, perhaps `settings.FILE_CHARSET if settings.is_overridden('FILE_CHARSET') else 'utf-8'` might work?

Reply:
Thanks for the review. You're right. The check with `hasattr()` didn't work properly and I replaced it with `is_overridden()` with the new commit. It seems to work well now.